code/modules/client/preference/preferences_mysql.dm

```diff
@@ -90,7 +90,7 @@
 					UI_style_alpha='[UI_style_alpha]',
 					be_role='[sanitizeSQL(list2params(be_special))]',
 					default_slot='[default_slot]',
-					toggles='[num2text(toggles, 9)]',
+					toggles='[num2text(toggles, Ceiling(log(10, (TOGGLES_TOTAL+1))))]',
```

Comment:
`round(log(TOGGLES_TOTAL, 10))+1`

Reply:
Oh wait Ceiling is a legit proc, my bad. Use that instead

Reply:
base comes first in byond log

Reply:
![image](https://user-images.githubusercontent.com/30060146/61257920-22332200-a741-11e9-843c-f7c39e557272.png)

Reply:
Wait, my further bad. The order of arguments on `log` is correct, as well. However, why are you adding 1 to TOGGLES_TOTAL?

Reply:
TOGGLES_TOTAL is 16777215 so I wanted to make it the nearest bit which is 16777216

Reply:
but I realize that doesn't matter as the result is the same